### DISCUSSION TITLE
Top panel

### DISCUSSION THREAD

**Discussion by FatFace68** [luisbocanegra/plasma-panel-colorizer#38] [Q&A] [answered]
Can I make the panel have full opacity when a window touches it?

**Comment by luisbocanegra** [comment]
Currently not. But I recall seeing a property that indicated if a window was touching the panel, will have to check again if it can be used for this purpose

**Reply by FatFace68** [reply]
Thanks for the (very) quick reply. Nothing that I can see in the settings is obvious. I noticed on Reddit that you said that this option would be in a future update. Any timeline on that? 

Loving the widget btw, Great work, sir.

**Reply by luisbocanegra** [reply]
For opaque on maximized you can create two presets, one opaque and other transparent, then assign them to autoload on normal and maximized:
![Screenshot_20240405_043154](https://github.com/luisbocanegra/plasma-panel-colorizer/assets/15076387/f1b20bb6-5c9a-4132-a2bc-9a1738721cb1)

![maximized-preset](https://github.com/luisbocanegra/plasma-panel-colorizer/assets/15076387/9fbae295-be32-4eac-b443-1cc9bafa5b96)

In the same auto loading tab I will add another "Window is touching panel" if such property exists in the panel and can be used reliably

**Reply by FatFace68** [reply]
I've changed the presets but what else do I need to do/add? Sorry if I'm being dim about this, but nothing is leaping out at me in the settings. 

![Screenshot_20240405_154147](https://github.com/luisbocanegra/plasma-panel-colorizer/assets/166118116/afa967f8-5e90-4f0f-8b75-be047da963d5)

**Reply by luisbocanegra** [reply, marked as answer]
I think you are forgetting to update each preset after you change it:
![image](https://github.com/luisbocanegra/plasma-panel-colorizer/assets/15076387/e937bd9e-08ce-4dbc-bb54-297163f62ecd)

1. First you save a preset and name it wathever you lile e.g. transparent
2. Go to a tab you want to change e.g. Widget Background > Uncheck enable on top left corner
3. Go to another tab you want to change e.g. Panel background and check Hide under Original background section
4. Go back to General tab and update the profile you created in step 1 by clicking on the save button
5. Do again steps 1-4 but this time setting full opacity

**Reply by FatFace68** [reply]
Thank you very much for this. I'll give it a go later on.

You're a star

Cheers.

**Reply by FatFace68** [reply]
Right, now I understand the mistakes I was making, it's now working like a charm. 

Thank you again.
